portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random fact to the page.
- */
-function addRandomFact() {
-  const facts =
-      ['Plays the violin!', 'Enjoys running', 'Likes flying drones!', 'Plays slither.io'];
-
-  // Pick a random fact.
-  const fact = facts[Math.floor(Math.random() * facts.length)];
-
-  // Add it to the page.
-  const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
+function changeImage() {
+  document.getElementById("switch").src = "/images/hi.jpg";
+  document.getElementById("switch").style = "width:50%; padding-top:3em; padding-bottom:3em";
```

Comment:
Could we add these styles to the #switch css or make a class for this?

Reply:
Converted to a class